.circleci/config.yml

```diff
@@ -93,37 +93,41 @@ jobs:
       is-canary:
         type: string
         default: ""
-    docker:
-    - image: circleci/android:api-29-node@sha256:71d61d6c21b29948d57120f476a83cc322a280979bce355c5a0ad771293ca380
-    environment:
-      JAVA_OPTS: '-Xms512m -Xmx2g'
-      GRADLE_OPTS: '-Xmx3g -Dorg.gradle.daemon=false -Dorg.gradle.jvmargs="-Xmx2g -XX:+HeapDumpOnOutOfMemoryError"'
+    machine:
+      image: ubuntu-1604:202007-01
     steps:
       - checkout
       - checkout-submodules
+      - run: node -v
+      - run: npm install -g yarn
       - npm-install
-      - add-jest-reporter-dir
+      - run: npm run test:e2e:bundle:android
       - run:
-          name: Set Environment Variables
+          name: Build apk
           command: |
-            echo 'export TEST_RN_PLATFORM=android' >> $BASH_ENV
-            echo 'export TEST_ENV=sauce' >> $BASH_ENV
-      - run:
-          name: Bundle Android and Generate debug .apk file for testing
-          command: npm run test:e2e:bundle:android && npm run core test:e2e:build-app:android
+            docker run --rm -it \
+            --volume $(pwd):/home/circleci/project \
+            --workdir /home/circleci/project \
+            --env JAVA_OPTS='-Xms512m -Xmx2g' \
+            --env GRADLE_OPTS='-Xmx3g -Dorg.gradle.daemon=false -Dorg.gradle.jvmargs="-Xmx2g -XX:+HeapDumpOnOutOfMemoryError"' \
+            circleci/android@sha256:71d61d6c21b29948d57120f476a83cc322a280979bce355c5a0ad771293ca380 \
```

Comment:
Not a blocker but, looks like we are referencing this docker image in a couple of places (here and in line 146) and I'm thinking if it makes sense to have a constant defined somewhere with this name, to avoid potentially changing the one but not the other?

Reply:
Good idea! But on second thought maybe it's not necessary to pin this image (which was done to pin node version) as we might not need node when building the apk. Let me try to remove the hash instead.